### PR TITLE
refactor terminator

### PR DESCRIPTION
**Pull request by loomis**
Refactor the terminator to avoid loading all user objects for every garbage collection cycle.  Add some logging and rationalize others.

### CODE REVIEW

File: rpm/src/main/jetty-config/resources/logging.properties [hunk 1]
```diff
@@ -17,14 +17,15 @@ java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
 
 # Mute org.restlet.ext.httpclient.internal.HttpMethodCall sendRequest
 java.util.logging.FileHandler.filter=com.sixsq.slipstream.util.RestletSendRequestMuteFilter
+java.util.logging.FileHandler.filter=
```
Comment (schaubl) [inline rpm/src/main/jetty-config/resources/logging.properties]: What is the rationale behind this change ?

Comment (loomis) [reply rpm/src/main/jetty-config/resources/logging.properties]: Good catch.  Cut & paste error.
